src/alarm-applet.c

```diff
@@ -576,42 +586,55 @@ alarm_applet_init (int *argc, char ***argv)
 	GOptionContext *context;
 
 	gboolean hidden = FALSE;     // Start hidden
+	gboolean stop_all = FALSE;   // Stop all alarms
+	gboolean snooze_all = FALSE; // Snooze all alarms
 
     // Command line options
 	GOptionEntry entries[] =
 	{
 		{ "hidden", 0, 0, G_OPTION_ARG_NONE, &hidden, "Start hidden", NULL },
+		{ "stop-all", 0, 0, G_OPTION_ARG_NONE, &stop_all, "Stop all alarms",
+			NULL },
+		{ "snooze-all", 0, 0, G_OPTION_ARG_NONE, &snooze_all,
+			"Snooze all alarms", NULL },
 		{ NULL }
 	};
 
+	// Parse command-line arguments
+	context = g_option_context_new (NULL);
+	g_option_context_add_main_entries (context, entries, GETTEXT_PACKAGE);
+	g_option_context_add_group (context, gtk_get_option_group (TRUE));
+
+	if (!g_option_context_parse (context, argc, argv, &error)) {
+		g_printerr ("%s\n", error->message);
+		exit (EXIT_FAILURE);
+	}
+
 	// Initialize unique app
 	unique_app = unique_app_new ("com.pseudoberries.AlarmClock", NULL);
+	unique_app_add_command(unique_app, "stop all alarms", UNIQUE_STOP_ALL);
```

Comment:
In libunique, where is the command_name used? Is it appropriate with spaces in the command name?

Reply:
I'm not sure. In the code of `unique_app_add_command()`, it's just said for internal use. And the name was stored in hash tables as key and also value.

Anyway, I changed the command names to CamelCase to be safe.